scripts/mongosh/init_aasd_db.js

```diff
@@ -7,3 +7,11 @@ db.jobOffers.insertOne({
     "status": 0,
     "applications": []
 });
+
+db.recruitmentInstructions.insertOne({
+    "_id": ObjectId("674aff48c8c935e125c43951"),
+    "job_offer_id": ObjectId("674aff48c8c935e125c43939"),
+    "stage_number": 3,
+    "stage_types": [1, 2, 3],
+    "stage_priorities": [1, 1, 2]
```

Comment:
na pierwszy rzut oka wydaje mi się że stage_number jest zbędny bo można wydedukować z innych pól. Ale jeśli wymaga to zmian w wielu miejscach to już tak zostaw

Reply:
moge wywalic